types/MimicryMacroMarketMashup.md

```diff
@@ -0,0 +1,211 @@
+## Type Name
+
+- `MimicryMacroMarketMashup`
+
+
+## Description
+
+This query returns a single market capitalization metric derived from the sum of zero or more NFT collection market capitialization metrics, plus zero or more ERC-20 market capitalization metrics.
+
+Its initial use case is enabling the creation of macro-markets on [Mimicry Protocol](https://www.mimicry.org/), but the values can be used by anyone who needs this metric.
+
+### Market Capitalization for NFT Collections, Explained
+Market capitalization is calculated as the sum of each NFT within an NFT collection, where each NFT is valued at the greater of its last traded price or the floor price of the collection, respectively.
+
+#### For ERC721 collections:
+- Market Cap = the sum of all NFT values of the collection
+- The value of each NFT = max(last price, floor price)
+
+#### For ERC1155 collections:
+- Market Cap = the sum of all NFT values of the collection
+- The value of each NFT = max(last price, floor price) times the number of owners
+
+Suspected wash trades should be filtered out.
+
+> Note that this market capitalization methodology has been heavily influenced by NFTGo's [collection metrics documentation](https://docs.nftgo.io/docs/collection-metrics). 
+
+### Market Capitalization for ERC-20 Tokens, Explained
+The market capitalization of an ERC-20 token is the total market value of a cryptocurrency's circulating supply. It is analogous to the free-float capitalization in the stock market.
+
+- Market Cap = Current Price * Circulating Supply
+
+
+## Query Parameters
+
+The `MimicryMacroMarketMashup` query has two parameters:
+
+1. **ethereumNftCollections**
+    - description: A CSV list of NFT collection contract-addresses on the Ethereum Mainnet
+    - value type: `string`
+3. **tokens**
+    - description: A CSV list of cryptocurrency tokens (e.g. 'APE, MANA, SAND')
```

Comment:
and this would be an array of token symbols or token names?  (or just identifiers from some other source (e.g. coinGecko)). so type "string[]"